src/app/views/communities/view/overview/overview.vue

```diff
@@ -22,6 +22,47 @@
 					<br />
 					<app-game-thumbnail :game="community.game" class="-community-game" />
 				</template>
+
+				<div v-if="shouldShowCollabSection" class="-mod-list">
+					<div class="clearfix">
+						<div class="pull-right" v-if="shouldShowLoadMoreCollaborators">
+							<app-button
+								trans
+								:disabled="isLoadingMoreCollaborators"
+								@click="toggleCollaboratorList"
+							>
+								<translate v-if="collaboratorListCollapsed || isLoadingMoreCollaborators || hasMoreCollaborators">View All</translate>
+								<translate v-else>Show fewer</translate>
+							</app-button>
+						</div>
+
+						<h5 class="section-header">
+							<translate>Moderators</translate>
+						</h5>
+					</div>
+
+					<div v-for="user of moderators" :key="user.id" class="-mod-list-entry">
+						<app-user-card-hover :user="user">
+							<router-link :to="user.url">
+								<span>
+									{{ user.display_name }}
```

Comment:
Use the username instead to make it easier for users to address and mention the moderators if needed.